lib/registries/addon/components/registries-registration-type-facet/component.ts

```diff
@@ -43,11 +36,14 @@ export default class RegistriesRegistrationTypeFacet extends Component {
     @waitFor
     async fetchRegistrationTypes() {
         try {
-            const metaschemas = await this.store.query('registration-schema', {
+            if (!this.provider){
+                this.provider = await this.store.findRecord('registration-provider', 'osf');
```

Comment:
In lib/registries/config/environment.js there is a `defaultProviderId` that we could use.